### ISSUE TITLE
This error pops up and vr won't start, please help.

### ISSUE THREAD

**Issue by MintiiVanilla** [DaXcess/LCVR#53]
![image](https://github.com/DaXcess/LCVR/assets/156265923/fde48ee6-13d0-45fc-a262-0be63fda6a86)

Please, i'm getting a headache from trying to figure this out and would really appreciate you lovely people helping.

**Comment by MintiiVanilla** [comment]
> Incompatible with latest Lethal Company Update.

Forgot to mention this is even after i reverted

**Comment by JTMCDev** [comment]
DaXcess should give me power to close issues.

This is irrelevant. Please wait for releases or make sure that when your on the main menu it says V45 on the bottom left corner.

**Comment by Oslott1** [comment]
I honestly don't understand why people are having issues running the mod on the updated version of the game... I've had no issues with it (except 2 bugs I've noticed mainly, double menu screen and player arms animating unnaturally). Running it with 15 other mods too. The only time I had issues launching was when the game updated and I launched without rebuilding the project. When the game updates, many mods receive updates and the files are changed, right? So update your mod and make sure you also replace/update Assembly-CSharp.dll and DissonanceVoip.dll in the repo with the updated ones from the local game files before you build! Then just update the LCVR.dll in your plugins folder with the new one you just built!

Also, because DaXcess is a great dev, DaXcess included a version check function that verifies the game version is supported by the mod, insuring the mod always runs smoothly and that the user's experience is as enjoyable as possible! To launch the game with your updated DLL, you must add --lcvr-skip-shasum to the game's launch parameters. This will bypass the function and allow you to launch the game!

Hope this helps everyone solve this problem!
